packages/admin/src/components/MenuTree/MenuTree.js

```diff
@@ -31,7 +14,7 @@ const MenuItem = ({item, typeMapping}) => {
   }
 
   const Component = mappedType.component
-  return <StyledMenuEntry {...item}>
+  return <StyledMenuEntry {...item} childrenCount={item.children.length}>
```

Comment:
I checked it. item can never be undefined. nevertheless i would add a check. Otherwise someone might refactor a part and we get a error here.